data/json/items/armor/storage.json

```diff
@@ -515,6 +515,7 @@
     "covers": [ "LEGS" ],
     "coverage": 20,
     "encumbrance": 5,
+    "pocket_data": [ { "pocket_type": "CONTAINER", "max_contains_volume": "2250 ml", "max_contains_weight": "2 kg", "moves": 200 } ],
```

Comment:
shouldn't this be 2 pockets? one for each leg?

Reply:
Oh, yes you're right. I've updated to reflect that